docs/configuration.rst

```diff
@@ -217,6 +231,14 @@ The ``type`` field in the ``crazyflies.yaml`` entries must refer to a type liste
       #. Find the coordinates of the used markers, for example by using ``roslaunch crazyswarm mocap_helper.launch``. (You may need to do ``source ros_ws/devel/setup.bash`` before ``roslaunch``)
       #. Update ``crazyflieTypes.yaml``.
 
+   .. group-tab:: Single Marker
+
+      For single markers, the ``markerConfigurations`` entry simply contains a single ``points`` entry. This point should describe the offset of the marker with respect to the Crazyflie's center of mass. For example, the marker configuration ``"3"`` corresponds to the marker placement shown below:
```

Comment:
Did you mean to add a third `markerConfigurations` entry to the example `crazyflieTypes.yaml`?

Reply:
No, this is already in there for a while - it was simply not documented (apart from a comment in `crazyflieTypes.yaml`).